nextflow_schema.json

```diff
@@ -107,51 +107,67 @@
         "profile_dists": {
             "title": "Profile Dists",
             "type": "object",
-            "description": "",
+            "description": "Parameters for profile_dists distance calculations",
             "default": "",
             "properties": {
                 "pd_outfmt": {
                     "type": "string",
+                    "description": "The output format for distances",
+                    "enum": ["matrix", "pairwise"],
                     "default": "matrix"
                 },
                 "pd_distm": {
                     "type": "string",
+                    "description": "The distance method/unit",
+                    "enum": ["hamming", "scaled"],
                     "default": "scaled"
                 },
                 "pd_missing_threshold": {
                     "type": "number",
-                    "default": 1.0
+                    "description": "The maximum proportion of missing data per locus",
+                    "minimum": 0,
+                    "maximum": 1,
+                    "default": 1
                 },
                 "pd_sample_quality_threshold": {
                     "type": "number",
-                    "default": 1.0
+                    "description": "The maximum proportion of missing data per sample",
+                    "minimum": 0,
+                    "maximum": 1,
+                    "default": 1
                 },
                 "pd_match_threshold": {
                     "type": "number",
-                    "default": -1.0
+                    "description": "Threshold for matches. Should match unit used in pd_distm. Used only when pd_outfmt is set to pairwise.",
+                    "default": -1
                 },
                 "pd_file_type": {
                     "type": "string",
+                    "description": "Output format file type",
+                    "enum": ["text", "parquet"],
                     "default": "text"
                 },
                 "pd_mapping_file": {
                     "type": "string",
-                    "pattern": "^\\S+$",
+                    "pattern": "^\\S+\\.json(\\.gz)?$",
+                    "description": "Allele mapping file",
+                    "exists": true,
                     "format": "file-path"
                 },
-                "pd_force": {
-                    "type": "boolean"
-                },
                 "pd_skip": {
+                    "description": "Skip QA/QC steps",
                     "type": "boolean"
                 },
                 "pd_columns": {
                     "type": "string",
                     "pattern": "^\\S+$",
+                    "description": "Single column file with one column name per line or list of columns comma separate",
```

Comment:
As before, not sure what these columns are for. Maybe someone familiar with the problem would know?

Reply:
Added more information here 89e277754c85dcb018bde31f4bd7d5fed95f5dbb